tests/dummy/app/templates/application.hbs

```diff
@@ -8,6 +8,18 @@
   <div class="acceptance-feature-off"></div>
 {{/if-feature}}
 
+{{#if-feature 'feature-from-config'}}
+  <div class="feature-from-config-on"></div>
+{{else}}
+  <div class="feature-from-config-off"></div>
+{{/if-feature}}
+
 {{#if-feature 'off-feature'}}
   <div class="off-feature-on"></div>
 {{/if-feature}}
+
+{{#if features.flags.acceptance-feature}}
```

Comment:
It is used in the _visiting / with acceptance-feature on_ test that refers to `.bound-acceptance-feature-on`.

Uh, okay. Have removed.